betovarg/lucysanchez
# LS-4: homepage hero

## Steps to test

- go the the repository branch feature/LS-4-homepage-hero
- clean caches of Grav on admin Grav. ---> bin/grav clear-cache
- go to localhost:8080/
- review if the Hero module appears
- be free of modify things on the Grav admin to check if actually changes hero
